network/local.go

```diff
@@ -41,6 +41,8 @@ type LocalManager struct {
 	// close on this channel indicates that connection retries should stop
 	stopping chan bool
 	stopOnce sync.Once
+	// a waitgroup to check that all serving goroutines are done
```

Comment:
Does that change belong into the current PR?

Reply:
Without this, the builds were not passing, so no it's not related, but yes it's needed.